Sources/Rendering/Core/AbstractMapper3D/index.js

```diff
@@ -41,10 +30,10 @@ function vtkAbstractMapper3D(publicAPI, model) {
 // ----------------------------------------------------------------------------
 
 const DEFAULT_VALUES = {
-  bounds: [1, -1, 1, -1, 1, -1],
+  bounds: [...vtkBoundingBox.INIT_BOUNDS],
   center: [0, 0, 0],
 
-  viewSpecificProperties: null,
+  viewSpecificProperties: {},
```

Comment:
this is not equivalent to initializing it in `extend()`.

Here ALL vtkAbstractMapper instanes will share the same `viewSpecificProperties`.

You can fix it by using the `defaultValues()` function notation.